packages/renderer/src/lib/service/ServiceActions.spec.ts

```diff
@@ -51,7 +52,7 @@ class ServiceUIImpl {
   }
 }
 
-const service: ServiceUI = new ServiceUIImpl('my-service', 'RUNNING', '', false, '', '', '');
+const service: ServiceUI = new ServiceUIImpl('123', 'my-service', 'RUNNING', '', false, '', '', '');
```

Comment:
non-blocking enhancement: it starts to be a very long constructor, maybe need to have an object like {uid:'123', ...}